api/src/main/java/com/exadel/frs/core/trainservice/dao/FaceDao.java

```diff
@@ -73,13 +73,19 @@ public Face addNewFace(
                 .setEmbedding(embeddings)
                 .setFaceName(faceName)
                 .setApiKey(modelKey);
-        val image = new Image()
-                .setFaceImg(file.getBytes())
-                .setRawImg(file.getBytes())
-                .setFace(face);
 
-        imagesRepository.save(image);
+        val saveImagesToDb = Boolean.parseBoolean(
+                System.getenv().get("SAVE_IMAGES_TO_DB"));
```

Comment:
Please add default value as a constant in case user want to run this container not from docker-compose and forgot to add environment variable. 
The default value is true